wpimath/src/main/native/include/frc/geometry/Twist2d.h

```diff
@@ -35,6 +37,25 @@ struct WPILIB_DLLEXPORT Twist2d {
    */
   units::radian_t dtheta = 0_rad;
 
+
+  /**
+   * Obtain a new Transform2d from a (constant curvature) velocity.
+   *
+   * <p>See <a href="https://file.tavsys.net/control/controls-engineering-in-frc.pdf">Controls
```

Comment:
JavaDoc needs `<p>` tags in .java files, but Doxygen doesn't need `<p>` tags in C++ files, so they should be removed from the latter.